src/sentry/mail/notifications.py

```diff
@@ -79,6 +82,9 @@ def log_message(notification: BaseNotification, user: User) -> None:
                 "target_identifier": notification.target_identifier,
             }
         )
+        if len(notification.rules):
+            extra.update({"rule_id": notification.rules[0].id})
```

Comment:
Is this being added for logging?

Reply:
Yeah the code previously had
```
for future in futures:
  rules.append(future.rule)
  extra["rule_id"] = future.rule.id
  ...
```
so I just took the value from the `rules` array.